common/lib/xmodule/xmodule/templates/problem/checkboxes_response_hint.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
 
       Use the following example problem as a model.
 
-      >>Which of the following is a fruit? Check all that apply.<<
+      >>Which of the following is a fruit? Check all that apply.||Make sure you select all of the correct options—there may be more than one!<<
```

Comment:
I don't think we need the text "Check all that apply" here since we specify that in the description text.